thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomalydetection/performanceEvaluation/AnomalyPercentagePerformanceEvaluation.java

```diff
@@ -0,0 +1,27 @@
+package com.linkedin.thirdeye.anomalydetection.performanceEvaluation;
+
+import com.linkedin.thirdeye.datalayer.bao.MergedAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import java.util.List;
+import org.joda.time.Interval;
+
+public class AnomalyPercentagePerformanceEvaluation implements PerformanceEvaluate{
+  private Interval windowInterval;
+  private List<MergedAnomalyResultDTO> detectedResults;
+
+  public AnomalyPercentagePerformanceEvaluation(Interval windowInterval, List<MergedAnomalyResultDTO> detectedResults) {
+    this.windowInterval = windowInterval;
+    this.detectedResults = detectedResults;
+  }
+
+  @Override
+  public double evaluate(){
+    long anomalyLength = 0;
+    long totalLength = windowInterval.toDurationMillis();
+    for(MergedAnomalyResultDTO mergedAnomaly : detectedResults){
```

Comment:
Here you assume that the anomaly result in the list do not have overlap, this is not always true. Because the data structure itself dose not require the elements not overlapping. Or you should provide better comments/descriptions here to show the proper way of using this class/method.